### DISCUSSION TITLE
guides/

### DISCUSSION THREAD

**Discussion by giscus** [odpi/egeria-docs#822] [DocFeedback]
# guides/

Open metadata and governance for enterprises - automatically capturing, managing and exchanging metadata between tools and platforms

https://egeria-project.org/guides/

**Comment by srehmann** [comment]
Please fix the typo in the Using Egeria section. Below is the typo:

"Usiing Egeria should enable your organization to change the way it works, by improving the collaboration and shared understanding of your data and digital resources."

Using is written with double i.

(A small correction)

**Reply by mandy-chessell** [reply]
Thanks for spotting this ...
